coding/cpp_style_guide.rst

```diff
@@ -2007,6 +2006,15 @@ Because ``override`` already implies ``virtual``, virtual should not be included
 6.1. Layout
 -----------
 
+.. _style-guide-cpp-6-0:
+
+6-0. Layout MAY be automated with clang-format.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The easiest way to comply with the :ref:`layout rules <style-guide-cpp-layout>` is to use `clang-format <http://clang.llvm.org/docs/ClangFormat.html>`_.
```

Comment:
Consider adding something like "The intent is that all of the layout rules are compatible with clang-format using the LSST configuration; if any conflicts arise, this rule ensures that the clang-format result is permitted to be used notwithstanding any other."